CountryPickeriOS13+/CountryPickerSPM/Sources/CountryPickerSPM/Constant.swift

```diff
@@ -0,0 +1,265 @@
+//
+//  Constant.swift
+//  CountryCodeInSwift3
+//
+//  Created by Cl-macmini-100 on 12/29/16.
+//  Copyright © 2016 Suryakant. All rights reserved.
+//
+
+import Foundation
+
+// ISO 3166-1 alpha-2 two-letter country codes.
+let isoToDigitCountryCodeDictionary: [String: String] = [
```

Comment:
I think it is better to put computed property under some construct instead of keeping it open.

Reply:
hmm I think it is ok because it is internal to sdk